### ISSUE TITLE
Enhancement request: Create a .github/ISSUE_TEMPLATE here

### ISSUE THREAD

**Issue by texadactyl** [Legrandin/pycryptodome#417]
See https://github.com/UCBerkeleySETI/blimpy for an example.

**Comment by texadactyl** [comment]
Avoid future incoherent brain dumps!  (=:

**Comment by texadactyl** [comment]
You also might like to have a look at blimpy's CODE_OF_CONDUCT.md file.

**Comment by Varbin** [comment]
I agree fully. I suggest adding some checkboxes for the reporter to fill out (e.g. did you search for similar issues, read documentation, etc.), some specific tasks to do (e.g. commands to show the version number of Python, pycryptodome and so on) and areas to fill in code and output so reporters not familiar with markdown can make formatted reports.
